### ISSUE TITLE
500 error on creation finding with Jira link

### ISSUE THREAD

**Issue by zakrush** [DefectDojo/django-DefectDojo#4684]
**Be informative**
DD is v 1.15.0  into Docker. 
Integrated with jira

**Bug description**
I have Server Error (500) into /product/94/ad_hoc_finding, when I try to create finding with linked Jira Issue. 

If I create Finding without  `Linked JIRA Issue`, and then edit finding(setup Jira Issue) all are working fine.
**Steps to reproduce**
1. Go to Product that has Jira integration
2. Create Issue into Jira (ISS-1 for example)
2. Findings - Create new Finding 
3. Setup Linked JIRA Issue as ISS-1
4. Click Finish
5. See 500 ERROR

**Expected behavior**
Creating new finding with jira linked

**Deployment method** *(select with an `X`)*
- [ ] Kubernetes
- [`X` ] Docker
- [ ] setup.bash / legacy-setup.bash

**Environment information**
 - Operating System: Ubuntu 20.04
 - DefectDojo Commit Message: [use `git show -s --format="[%ci] %h: %s [%d]"`]

**Sample scan files** (optional)
If applicable, add sample scan files to help reproduce your problem.

**Screenshots** (optional)

**Console logs** (optional)
If applicable, add console logs to help explain your problem.

**Additional context** (optional)

**Comment by valentijnscholten** [comment]
What's in the logs? `docker-compose logs`

**Comment by zakrush** [comment]
Sorry, added logs:

```
uwsgi_1         | ERROR:django.request:Internal Server Error: /product/94/ad_hoc_finding
uwsgi_1         | Traceback (most recent call last):
uwsgi_1         |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
uwsgi_1         |     response = get_response(request)
uwsgi_1         |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 115, in _get_response
uwsgi_1         |     response = self.process_exception_by_middleware(e, request)
uwsgi_1         |   File "/usr/local/lib/python3.6/site-packages/django/core/handlers/base.py", line 113, in _get_response
uwsgi_1         |     response = wrapped_callback(request, *callback_args, **callback_kwargs)
uwsgi_1         |   File "./dojo/authorization/authorization_decorators.py", line 39, in _wrapped
uwsgi_1         |     return func(request, *args, **kwargs)
uwsgi_1         |   File "./dojo/product/views.py", line 1167, in ad_hoc_finding
uwsgi_1         |     if form.is_valid() and (jform is None or jform.is_valid()):
uwsgi_1         |   File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 185, in is_valid
uwsgi_1         |     return self.is_bound and not self.errors
uwsgi_1         |   File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 180, in errors
uwsgi_1         |     self.full_clean()
uwsgi_1         |   File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 382, in full_clean
uwsgi_1         |     self._clean_form()
uwsgi_1         |   File "/usr/local/lib/python3.6/site-packages/django/forms/forms.py", line 409, in _clean_form
uwsgi_1         |     cleaned_data = self.clean()
uwsgi_1         |   File "./dojo/forms.py", line 2345, in clean
uwsgi_1         |     if jira_issue_key_new and not finding.has_jira_group_issue:
uwsgi_1         | AttributeError: 'NoneType' object has no attribute 'has_jira_group_issue'
```

**Comment by valentijnscholten** [comment]
fixed PR

**Comment by valentijnscholten** [comment]
fixed in dev / 2.0.0
